src/main/java/plugily/projects/villagedefense/arena/ArenaEvents.java

```diff
@@ -272,11 +272,7 @@ public void run() {
           return;
         }
         if (user.isSpectator()) {
-          if (ServerVersion.Version.isCurrentEqualOrHigher(ServerVersion.Version.v1_16_R1)) {
-            user.getPlayer().sendActionBar(TextComponent.fromLegacyText(plugin.getChatManager().colorMessage(Messages.DIED_RESPAWN_IN_NEXT_WAVE)));
-          } else {
-            user.getPlayer().spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(plugin.getChatManager().colorMessage(Messages.DIED_RESPAWN_IN_NEXT_WAVE)));
-          }
+          user.getPlayer().spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(plugin.getChatManager().colorMessage(Messages.DIED_RESPAWN_IN_NEXT_WAVE)));
```

Comment:
Please use the Commons Box provided `MiscUtils#sendActionbar` method.

Reply:
Changed :D